tools/dds/dds-sniffer/rs-dds-sniffer.cpp

```diff
@@ -59,6 +64,38 @@ struct log_consumer : eprosima::fastdds::dds::LogConsumer
 };
 
 
+static eprosima::fastrtps::rtps::GuidPrefix_t std_prefix;
+std::string print( const librealsense::dds::dds_guid & guid, bool shorten = true )
+{
+    std::ostringstream output;
+    if( guid != eprosima::fastrtps::rtps::c_Guid_Unknown )
+    {
+        output << std::hex;
+        char old_fill = output.fill( '0' );
+        size_t i = 0;
+        while( i < std_prefix.size  &&  guid.guidPrefix.value[i] == std_prefix.value[i] )
```

Comment:
std_prefix is of this participant GUID. Most likely prints will be of entities outside this participant, so check is redundant.

Reply:
But the same vensor, or the same host.
This does save on screen clutter, especially in testing.